packages/ComponentLibrary/src/components/Drawer/DrawerSubsection/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { useCallback, useState } from 'react';
+import { Collapse, Box, Typography } from '@mui/material';
+import { ExpandLess, ExpandMore } from '@mui/icons-material';
+import { styles } from '../styles';
+import { theme } from '../../../theme';
+import { DrawerSubsectionProps } from '../types';
+
+const open = true;
+
+const DrawerSubsection = ({ item, onClick }: DrawerSubsectionProps) => {
+  const isMainSection = item.type === 'folder';
+  const isSelected = false;
+  const [expanded, setExpanded] = useState(false);
+
+  const handleClick = useCallback(() => {
+    if (item.type === 'folder') {
+      setExpanded(prev => !prev);
+    } else if (item.type === 'window') {
+      onClick(`/window/${item.windowId}`);
+    }
+  }, [item.type, item.windowId, onClick]);
+
+  return (
```

Comment:
having many applications of conditional styles is a bit difficult to read and maintain.

I would extrude styles to variables or functions, for example: 

const drawerSectionStyle = {
  ...styles.drawerSectionBox,
  backgroundColor: isMainSection && isSelected
    ? theme.palette.baselineColor.neutral[10]
    : 'transparent',
  padding: isMainSection && !!item.submenu && isSelected ? '0.5rem' : undefined,
};